trainingRN/src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createStackNavigator } from 'react-navigation';
+import Todo from './screens/todo';
+import BookList from './screens/bookList';
+import routes from './constants/routes';
+
+const RootStack = createStackNavigator(
+  {
+    [routes.HOME]: Todo,
+    [routes.BOOK_LIST]: BookList
```

Comment:
Use a TabNavigator for these two

Reply:
I'm using `stackNavigator` do you have an example on how to use TabNavigator?

Reply:
[Docs](https://reactnavigation.org/docs/en/tab-based-navigation.html) may help here 😊

Reply:
Okey, since this is the only comment here i will apply this change in the other PR with the book list